lms/djangoapps/course_home_api/outline/v1/serializers.py

```diff
@@ -3,6 +3,7 @@
 """
 
 from django.utils.translation import ngettext
+from django.conf import settings
```

Comment:
Am I correct that you're adding the `hash_key` to the URLs here so that the course outline can use them in their links?

If so, you might want to leave those links unchanged for now, and leave the course_home_api as-is. The old links should be fine to stay on the outline page for now, as they'll redirect to the new ones.

Reply:
I think that is why I am adding them. Not 100%.